spec/controllers/roles_controller_spec.rb

```diff
@@ -1,5 +1,46 @@
 require 'rails_helper'
 
 RSpec.describe RolesController, type: :controller do
+  describe "GET #index" do
```

Comment:
Avoid committing empty pending specs.

Pending specs can be useful, but they must still implement an expectation. That way RSpec will yell at you if the spec suddenly passes